client/src/app/App.js

```diff
@@ -1136,8 +1134,7 @@ export class App extends PureComponent {
     if (
       activeTab !== prevState.activeTab ||
       tabs !== prevState.tabs ||
-      layout !== prevState.layout ||
-      endpoints !== prevState.endpoints
```

Comment:
This will be used (again) once we support "endpoints" for deployment. 

Why would we remove it here?

Reply:
(Of course fine, if we consider to store the endpoints in a different place, once supported).